apps/condo/pages/division/[id]/index.tsx

```diff
@@ -76,7 +78,11 @@ export const DivisionPageContent = ({ division, loading, columns, role }: Divisi
             </Typography.Title>
             <Row gutter={[0, 20]}>
                 <PageFieldRow labelSpan={5} title={ResponsibleLabelMessage} highlight>
-                    {get(division, ['responsible', 'name'])}
+                    <Link href={`/employee/${responsible.id}`}>
```

Comment:
responsible can be null

Reply:
Responsible is required field
<img width="742" alt="image" src="https://user-images.githubusercontent.com/10697310/163331100-9f663f99-64cd-40e7-8639-9801f174e176.png">